rules/rules_test.go

```diff
@@ -551,7 +551,7 @@ func TestExpressions(t *testing.T) {
 			fullRanges:     0,
 			intervalRanges: 2,
 		},
-		{ // Round should round half to even.
+                {
```

Comment:
Seems like you introduced some spaces (vs. tabs) here. Could you run "make format"?